addons/events/script_component.hpp

```diff
@@ -31,6 +31,12 @@
 
 #define SEND_TEVENT_TO_SERVER(params,name,targets) TEVENT_PVAR = [name, params, targets]; publicVariableServer TEVENT_PVAR_STR
 
+// turret events
+#define TUEVENT_PVAR CBAr
```

Comment:
This is the most beautiful pull request I have ever seen, however, CBAr and CBAt are the predecessors of CBAs and CBAu and they were changed for reasons. Literally every other character is free, but not r and t.
```suggestion
#define TUEVENT_PVAR CBAv
```

Reply:
4 characters out of 26 that were not available, I managed to pick one. 😂

Reply:
That's why I told you which ones, because the next guess would've been CBAt for "turret".